src/main/java/seedu/address/logic/commands/EditClientCommand.java

```diff
@@ -94,9 +96,15 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_NON_EXISTING_PRODUCT);
         }
 
+
         model.setClient(clientToEdit, editedClient);
         model.updateFilteredClientList(PREDICATE_SHOW_ALL_CLIENTS);
-        return new CommandResult(String.format(MESSAGE_EDIT_CLIENT_SUCCESS, editedClient));
+        clientToEdit.getMeetings().forEach(meeting ->
```

Comment:
instead of shifting like this, why not just `model.setClient` with a editedClient with the correct meetings?